scripts/bug_retriever.py

```diff
@@ -17,26 +17,39 @@ class Retriever(object):
     def retrieve_bugs(self, limit: int = None) -> None:
         bugzilla.set_token(get_secret("BUGZILLA_TOKEN"))
 
+        last_modified = None
         db.download(bugzilla.BUGS_DB)
 
         # Get IDs of bugs changed since last run.
-        last_modified = db.last_modified(bugzilla.BUGS_DB)
-        logger.info(
-            f"Retrieving IDs of bugs modified since the last run on {last_modified}"
-        )
-        changed_ids = set(
-            bugzilla.get_ids(
-                {"f1": "delta_ts", "o1": "greaterthaneq", "v1": last_modified.date()}
+        try:
+            last_modified = db.last_modified(bugzilla.BUGS_DB)
+        except Exception as e:
+            if str(e) == "Last-Modified is not available":
+                pass
+
+        if last_modified:
+            logger.info(
+                f"Retrieving IDs of bugs modified since the last run on {last_modified}"
             )
-        )
+            changed_ids = set(
```

Comment:
Here you have the same problem you had before with `last_modified` being undefined.

If `last_modified` is not None, you enter this if and changed_ids has a value.
If `last_modified` is None, you don't enter this if and changed_ids doesn't have a value.
You should add an `else` to the `if last_modified` and set changed_ids to an empty set.

Reply:
Done